ospfd/ospf_lsa.c

```diff
@@ -2084,12 +2081,24 @@ void ospf_external_lsa_rid_change(struct ospf *ospf)
 					continue;
 
 				if (is_prefix_default(
-						(struct prefix_ipv4 *)&ei->p))
+					    (struct prefix_ipv4 *)&ei->p))
+					continue;
+
+				if (!ospf_redistribute_check(ospf, ei, NULL))
 					continue;
 
-				if (!ospf_external_lsa_originate(ospf, ei))
+				aggr = ospf_external_aggr_match(ospf, &ei->p);
+				if (aggr) {
+					if (IS_DEBUG_OSPF(lsa, EXTNL_LSA_AGGR))
+						zlog_debug(
+							"Originate Summary LSA after reset/rid change");
```

Comment:
rid/RID -- just to make it easier to relate to "router ID" :-)

Reply:
done